test/main.cpp

```diff
@@ -334,11 +401,17 @@ int main() {
     // 0xf1402573 = csrrs	10,0, CSR=0xf14
     inst = mavis_facade.makeInst(0xf1402573, 0);
     assert(inst != nullptr);
+
     cout << "line " << dec << __LINE__ << ": " << "DASM: 0xf1402573 = " << inst->dasmString() << endl;
 
     mavis::ExtractorDirectInfo ex_info ("add", {1, 2}, {3});
     inst = mavis_facade.makeInstDirectly(ex_info, 0);
+    
     assert(inst != nullptr);
+    for (const auto& [field, value] : special_fields_map_) {
```

Comment:
Clean up formatting - open bracket on newline and proper indentation on closing bracket.